SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -386,6 +399,51 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 	return ComponentInterest;
 }
 
+RPCsOnEntityCreation USpatialSender::PackQueuedRPCsForActor(TArray<TSharedRef<FPendingRPCParams>>* RPCList, AActor* Actor)
+{
+	RPCsOnEntityCreation QueuedRPCs;
+	check((RPCList != nullptr) && (Actor != nullptr));
+	if ((RPCList == nullptr) || (Actor == nullptr))
```

Comment:
Hmm I find this `if` and `check` redundant. I think we should just crash if we are incorrectly using this function so you can remove the `if` statement.